roles/migrationcontroller/templates/mig_rbac.yml.j2

```diff
@@ -3,7 +3,7 @@ apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: ClusterRole
 metadata:
   creationTimestamp: null
-  name: manager-role
+  name: migration-manager-role
```

Comment:
Thank you.... I saw 'manager-role' earlier today and made a note to come back and ask for this to be updated to something less generic for a cluster scoped entity